test/unit_tests/datasets/test_dataset.py

```diff
@@ -308,3 +308,25 @@ def test_set_description_windows_dataset(concat_windows_dataset):
         " True."
     ):
         window_ds.set_description(pd.Series({'wow': 'error'}), overwrite=False)
+
+
+def test_concat_dataset_get_sequence_out_of_range(concat_windows_dataset):
+    indices = [len(concat_windows_dataset)]
+    with pytest.raises(IndexError):
+        X, y = concat_windows_dataset[indices]
+
+
+def test_concat_dataset_seq_target_transform(concat_windows_dataset):
+    indices = range(100)
+    y = concat_windows_dataset[indices][1]
+
+    transform = lambda x: sum(x)  # noqa: E731
```

Comment:
a small thing, but why not just:
`transform=sum` ?
```suggestion
    transform =  sum
```

Reply:
Oops I simplified a more complicated expression and didn't realize what it had become :P